less/patternfly-react.less

```diff
@@ -1,3 +1,10 @@
 /**
   Patternfly React Specific Extensions
 */
+.label {
+  a.remove-button {
+    color: @color-pf-white;
+    display: inline-block;
+    margin-left: 5px;
+  }
+}
```

Comment:
This should be in  a separate partial file. Please see https://github.com/patternfly/patternfly-react/blob/master/CONTRIBUTING.md#adding-styling-for-your-components.

Please add `-pf` suffix to `remove-button` to avoid conflicts.